packages/document/docs/zh/api/config/config-frontmatter.mdx

```diff
@@ -57,6 +57,8 @@ export interface Hero {
   image?: {
     src: string;
     alt: string;
+    srcset?: string | string[];
+    sizes?: string | string[];
```

Comment:
can you add some examples about how to write `srcset` and `sizes` in docs below?

Reply:
```js
    /**
     * `srcset` and `sizes` are attributes of `<img>` tag. Please refer to https://mdn.io/srcset for the usage.
     * When the value is an array, rspress will join array members with commas.
     **/
```

i just simplily added this jsdoc for:

1. they are just html attributes of `<img>`, so i redirected a more comprehensive doc on MDN instead of duplicating the usage on our doc site.
2. they are advanced usage for those who need them, and  tyipcally also those who already know them
3. there's no elaboration and example for each property of `hero`, so we need a seperate pr to document it first

Reply:
LGTM，thanks for your contribution！